baremetal/metal3data_manager_test.go

```diff
@@ -1353,39 +1326,19 @@ var _ = Describe("Metal3Data manager", func() {
 				Expect(apierrors.IsNotFound(err)).To(BeTrue())
 			}
 		},
-		Entry("Already processed", testCaseReleaseAddressFromPool{
-			m3d: &infrav1.Metal3Data{
-				ObjectMeta: testObjectMeta(metal3DataName, namespaceName, ""),
-			},
-			poolName: testPoolName,
-			poolAddresses: map[string]bool{
-				testPoolName: true,
-			},
-			expectedAddresses: map[string]bool{
```

Comment:
I am thinking whats the harm if we keep this test, with the current code would it be a no-op ?

Reply:
It does not happen anymore. Each pool will only be processed once.

Reply:
ok. thanks for clarifying. Marking this comment as resolved.